index.tsx

```diff
@@ -84,15 +92,22 @@ function MasonryList<T>(props: Props<T>): ReactElement {
         if (isCloseToBottom(nativeEvent, onEndReachedThreshold || 0.1))
           onEndReached?.();
       }}>
-      {ListHeaderComponent}
+      <View style={ListHeaderComponentStyle}>{ListHeaderComponent}</View>
```

Comment:
I think this is `redundant`. Users can have more freedom to create components on the other side.

Reply:
The problem is when we want to apply style only for list, not for the header - we can't do that, thats why I think this props is should be, actually, for the same reason original FlatList in RN have this prop https://reactnative.dev/docs/flatlist#listheadercomponentstyle

Reply:
Thanks for the pointer! Totally understandable!